internal/service/credentials/resource_credential_type.go

```diff
@@ -118,6 +118,14 @@ func (r *CredentialTypeResource) Schema(ctx context.Context, req resource.Schema
 	const attrMaxPercent = 100
 	const imageMaxSize = 50000
 
+	titleDescription := framework.SchemaAttributeDescriptionFromMarkdown(
+		"Title of the credential. Verification sites are expected to be able to request the issued credential from the compatible wallet app using the title.  This value aligns to `${cardTItle}` in the `card_design_template`.",
```

Comment:
`${cardTItle}` regression

Reply:
I am very annoyed by this.  It's resolved.